### ISSUE TITLE
Clojure 1.9

### ISSUE THREAD

**Issue by include** [JonyEpsilon/gorilla-repl#280]
Hi,

Any plans to update Gorilla REPL to Clojure 1.9?
Is it anything a newbie can play with?

Cheers,
Francisco

**Comment by missingfaktor** [comment]
Hello! Could we have a release please? Pretty please. :-)

**Comment by glts** [comment]
Workaround for me was using a local build with (only gently) bumped deps versions.

To save you some time, here are the steps that worked for me (copy and paste each item into your shell).

# Install Gorilla REPL snapshot

1.  `git clone https://github.com/JonyEpsilon/gorilla-repl.git`
2.  `pushd gorilla-repl`
3.  ```sh
    patch -p1 <<EOF
    diff --git a/project.clj b/project.clj
    index e9e725a..f5b410c 100644
    --- a/project.clj
    +++ b/project.clj
    @@ -2,22 +2,23 @@
     ;;;;
     ;;;; gorilla-repl is licenced to you under the MIT licence. See the file LICENCE.txt for full details.
     
    -(defproject gorilla-repl "0.4.0"
    +(defproject gorilla-repl "0.4.1-SNAPSHOT"
       :description "A rich REPL for Clojure in the notebook style."
       :url "https://github.com/JonyEpsilon/gorilla-repl"
       :license {:name "MIT"}
    -  :dependencies ^:replace [[org.clojure/clojure "1.6.0"]
    -                           [http-kit "2.1.18"]
    +  :dependencies ^:replace [[org.clojure/clojure "1.8.0"]
    +                           [http-kit "2.2.0"]
                                [ring/ring-json "0.4.0"]
    +                           [ring/ring-core "1.6.3"]  ;; override!
                                [cheshire "5.3.1"]
                                [compojure "1.1.8"]
                                [org.slf4j/slf4j-api "1.7.7"]
                                [ch.qos.logback/logback-classic "1.1.2"]
                                [gorilla-renderable "2.0.0"]
                                [gorilla-plot "0.1.4"]
                                [javax.servlet/servlet-api "2.5"]
                                [grimradical/clj-semver "0.2.0" :exclusions [org.clojure/clojure]]
    -                           [cider/cider-nrepl "0.10.2"]
    +                           [cider/cider-nrepl "0.16.0"]
                                [org.clojure/tools.nrepl "0.2.12"]]
       :main ^:skip-aot gorilla-repl.core
       :target-path "target/%s")
    EOF
    ```
4.  `lein install`
5.  `popd`

# Install Gorilla REPL Leiningen plugin snapshot

1.  `git clone https://github.com/JonyEpsilon/lein-gorilla.git`
2.  `pushd lein-gorilla`
3.  ```sh
    patch -p1 <<EOF
    diff --git a/project.clj b/project.clj
    index c085154..54a1774 100644
    --- a/project.clj
    +++ b/project.clj
    @@ -1,6 +1,6 @@
    -(defproject lein-gorilla "0.4.0"
    +(defproject lein-gorilla "0.4.1-SNAPSHOT"
       :description "A Leiningen plugin for the Gorilla REPL."
       :url "https://github.com/JonyEpsilon/lein-gorilla"
       :license {:name "MIT"}
    -  :dependencies [[gorilla-repl "0.4.0"]]
    +  :dependencies [[gorilla-repl "0.4.1-SNAPSHOT"]]
       :eval-in-leiningen true)
    diff --git a/src/leiningen/gorilla.clj b/src/leiningen/gorilla.clj
    index 885091b..3ec2672 100644
    --- a/src/leiningen/gorilla.clj
    +++ b/src/leiningen/gorilla.clj
    @@ -10,7 +10,7 @@
                 [clojure.pprint :as pp]))
     
     ;; The version of Gorilla that we will use
    -(def gorilla-version "0.4.0")
    +(def gorilla-version "0.4.1-SNAPSHOT")
     
     ;; This is the leiningen task. It needs no arguments, and can run outside a project (assuming you've got the plugin
     ;; installed in your profile).
    EOF
    ```
4.  `lein install`
5.  `popd`

After this you should have a fresh Gorilla REPL 0.4.1-SNAPSHOT installed in your local repository and should be able to use it in the usual fashion (add `:plugins [[lein-gorilla "0.4.1-SNAPSHOT"]]` in the project and run `lein gorilla`).

**Comment by benfb** [comment]
Hey all,

Jony is busy right now and I'm finishing up my senior thesis. I'll have more time to work on this soon! Thanks for your patience, and the snapshot version should work for now.

**Comment by eddy147** [comment]
Any status update on this?

**Comment by benfb** [comment]
Hey! I finished my senior thesis and I've been thinking about continuing work on this project--I have to do some ground work on setting up some new infrastructure, but after that's set up we should be able to release more often. Sorry again for the delay!

**Comment by benfb** [comment]
A new release should go out within the next few days that should fix this. Thanks for waiting! I'll update this thread when it's out and you can let me know if there are any issues.

**Comment by benfb** [comment]
I just pushed the latest release of gorilla and the Lein plugin to a different repository/fork (`org.clojars.benfb/gorilla-repl`). The code for that plugin is located here: https://github.com/benfb/lein-gorilla. Jony has been mostly inactive, but I didn't want to make code changes without his approval under his name, so I created the other repository. Please raise any issues you have there. Thanks!

The newly-released version (0.5.0) of the forked plugin should fix this. To use it, just change the `lein-gorilla "0.4.0"` plugin to `org.clojars.benfb/lein-gorilla "0.5.0"`.
